vello/src/debug/validate.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use {
+    bytemuck::{Pod, Zeroable},
+    std::{collections::BTreeSet, fmt},
+    vello_encoding::LineSoup,
+};
+
+#[derive(Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord, Pod, Zeroable)]
+#[repr(C)]
+pub struct LineEndpoint {
+    pub path_ix: u32,
+
+    // Coordinates in IEEE-754 32-bit float representation
```

Comment:
why not just store them as `f32` ? (This should at least be explained here if it stays this way.)

Reply:
Added my best-guess explanation